go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/CosmWasm/wasmd v0.29.0
 	github.com/armon/go-metrics v0.4.1
 	github.com/btcsuite/btcd v0.22.3
-	github.com/cosmos/cosmos-proto v1.0.0-alpha8
+	github.com/cosmos/cosmos-proto v1.0.0-beta.1
```

Comment:
This carries with it a breaking update to `third_party/proto/cosmos_proto/cosmos.proto`.

The new version removes a field:
```diff
$ git diff third_party/proto/cosmos_proto/cosmos.proto
diff --git a/third_party/proto/cosmos_proto/cosmos.proto b/third_party/proto/cosmos_proto/cosmos.proto
index 0a073fc3..5c63b86f 100644
--- a/third_party/proto/cosmos_proto/cosmos.proto
+++ b/third_party/proto/cosmos_proto/cosmos.proto
@@ -88,17 +88,6 @@ message ScalarDescriptor {
     // encoding standards and simple and clear. Currently only string and
     // bytes fields are supported for scalars.
     repeated ScalarType field_type = 3;
-
-    // legacy_amino_encoding is an optional string to describe the encoding
-    // format used by Amino. The field type is chosen to be a string so that
-    // the value can either be:
-    // - a machine-readable string, such as "base64", "bech32" or "utf8",
-    // - or a human-readable string, for instance a short specification of how
-    // a big integer would be encoded using Amino.
-    //
-    // If left empty, then the Amino encoding is expected to be the same as the
-    // Protobuf one.
-    string legacy_amino_encoding = 4;
 }
```